omero/sysadmins/cli/usergroup.rst

```diff
@@ -19,15 +19,15 @@ command::
 
 During the addition of the new user, you will need to specify the first and
 last name of the new user and their username as well as the groups the user
-belongs to. To add John Smith as a member of group 2 identified as jsmith,
+belongs to. To add John Smith as a member of the group named ``test-group`` identified as ``jsmith``,
 enter::
 
-    $ bin/omero user add jsmith John Smith 2
+    $ bin/omero user add jsmith John Smith --group-name test-group
```

Comment:
I do not think you need the ``--group-name`` flag here.

For example

``bin/omero user add zpasta zubni pasta test-dom`` 
worked fine for me just now on eel, where ``test-dom`` is the name of a group on eel

```
bin/omero group info test-dom
Using session b81 (user-6@eel.openmicroscopy.org:4064). Idle timeout: 10 min. Current group: private-1
 id   | name     | perms  | ldap  | # of owners | # of members 
------+----------+--------+-------+-------------+--------------
 1603 | test-dom | rwr--- | False | 0           | 1            
(1 row)
```

Reply:
mainly for clarity